### DISCUSSION TITLE
3D printed venturis

### DISCUSSION THREAD

**Discussion by DeeEmm** [DIY-Flow-Bench/DIY-Flow-Bench#94] [Flow Bench Design]
I want to add support for venturi style benches but not having much exposure to venturi bench setups need to get an understanding for the requirements. My plan is to design some 3D printable venturis that can be used to create a venturi style bench. 

I suspect that these venturis could also be used as calibration 'orifices' for other styles of benches. Effectively they work in the same manner as  any calibration orifice, having a maximum flow capability based on their dimensions. As 3D printing can generate sub-mm accuracy, 3D printable venturis might be a good solution for a universal calibration device, especially with higher flowing benches.

Discuss...

**Comment by DeeEmm** [comment]
If I can get an understanding of the design requirements I'll design a venturi that can be printed out. It would be good to get people to print this and test it on their benches so that we can get an understanding of the variation is flow figures that come back.

I believe that a similar thing has been done by PTS flow benches where they have shipped a set of orifice plates to various flow bench owners.

(I do make carburettor venturis but this is a bit of a different scale)

**Comment by DeeEmm** [comment]
Already available (at a cost)... https://www.shapeways.com/product/ZFQJ7UAL2/flow-venturi-27mm

The maths looks pretty straightforwards. It's very similar to using a orifice.

<img width="871" alt="Screen Shot 2021-08-28 at 8 31 50 pm" src="https://user-images.githubusercontent.com/3038710/131217800-8c5e864b-1055-4729-8825-81c18f2d5148.png">

The actual design does not seem super critical, you just need to know the two diameters. The big advantage of using a venturi being that you do not get the loss of flow that you do with an orifice plate. So your vac source is doing a lot less work, which means you get a higher reference pressure.

From what I have read the venturi style flow measurement is far superior but due to the construction is generally very expensive. With the availability of 3D printing that cost is massively reduced.

At any rate, now that I have the maths, I will implement it in the code.

**Comment by DeeEmm** [comment]
OK, So had a quick look this morning at how to make venturi's on Fusion 360. It's actually very simple...

<img width="1313" alt="Screen Shot 2021-08-29 at 10 52 38 am" src="https://user-images.githubusercontent.com/3038710/131235208-657f9b23-3ecf-43d1-be78-750e5b16d76e.png">

First of all you need to sketch a rectangle that represents half of the side profile, for example, this rectangle is 50 x 200, which will give a venturi of 100 x 200mm (Outer Dims). Then using the curve tool draw your venturi profile. 

<img width="1312" alt="Screen Shot 2021-08-29 at 10 53 00 am" src="https://user-images.githubusercontent.com/3038710/131235238-48625f5a-aa26-4987-abfd-9ebf3902327d.png">

Next up, finish the sketch and choose the 'revolve' tool. For the 'profile' select the venturi profile you just drew and for the 'axis' select the right hand side of the rectangle you drew in step 1

<img width="1313" alt="Screen Shot 2021-08-29 at 10 53 35 am" src="https://user-images.githubusercontent.com/3038710/131235263-ec4dbaa6-1bc5-428f-a5ea-0eaec258529c.png">

It will then create a solid around this axis...

<img width="1313" alt="Screen Shot 2021-08-29 at 10 54 33 am" src="https://user-images.githubusercontent.com/3038710/131235268-6b8e0ca7-34e0-4e53-ac20-5056f6ed4001.png">

<img width="1314" alt="Screen Shot 2021-08-29 at 10 54 49 am" src="https://user-images.githubusercontent.com/3038710/131235270-f1096ca5-0336-4fdd-b012-f5bda40e76ac.png">


Pretty simple really. 

Obviously for this to be of any use you would also need to add additional sockets / bores to each end so that it can be used with whatever sized pipe you have built your bench out of, but as a proof of concept it works and allows you to accurately plot and 3D print whatever size venturis you might need.

I think it would be an interesting test to compare the theoretical flow of a venturi against its actual flow. This is one of the determining factors in being able to use a replicated 3D part as a calibration tool. If it is possible to make an accurate enough 3D printed part it negates the need to have the venturis flow tested on a regular flow bench.

I think that theoretically (at least) this is possible, especially on larger diameters. This is simply because if we look at the accuracy of a domestic 3D printer (generally 0.1mm) and compare this to the deviation you would see in application you are only looking at a deviation of around 0.1 cfm

I used the following online calculator to calculate the deviation in flow relative to a 0.1mm change in diameter

https://www.efunda.com/formulae/fluids/venturi_flowmeter.cfm#calc

This means that it would be possible to draft a set of standard venturis for flow bench calibration that can be 3D printed by the end user and used to calibrate their benches without the requirement to get the orifices flow tested and still produce accurate results.

Likewise, it would be possible to print a set of venturi flow meters for someone who wanted to build a venturi style bench instead of an orifice bench or a MAF based bench and produce accurate measurements without needing to calibrate the bench.

I think I will take a look at designing a set of venturis and getting them flow tested to test this theory 

/DM

**Comment by DeeEmm** [comment]
Found some reference for venturi designs in the Hyperflow DIY bench instructions.

<img width="925" alt="Screen Shot 2021-09-22 at 2 15 46 pm" src="https://user-images.githubusercontent.com/3038710/134284957-28450f42-507e-4cf8-9b9c-a8ab36296af7.png">

Should be fairly trivial to convert these into 3D printed versions.

**Comment by KirikauKiwi** [comment]
This looks great, as you say a good way to easily verify a flow bench and generally easily repeatable and scaleable to larger bore/pipe sizes etc

**Comment by stefan63** [comment]
For flow simulation and testing this is nice: https://sim-flow.com/openfoam-gui/
I have tested it while trying to optimize fan ducts and it is not impossible difficult to work with.

**Reply by DeeEmm** [reply]
Looks like a useful tool. Could use this on manifold design.
